src/util/script.rs

```diff
@@ -19,6 +19,33 @@ pub fn get_script_asm(script: &Script) -> String {
     (&asm[7..asm.len() - 1]).to_string()
 }
 
+pub fn get_script_type(script: &Script) -> String {
+    let script_type = if script.is_empty() {
+        "empty"
+    } else if script.is_op_return() {
+        "op_return"
+    } else if script.is_p2pk() {
+        "p2pk"
+    } else if script.is_p2pkh() {
+        "p2pkh"
+    } else if script.is_p2sh() {
+        "p2sh"
+    } else if script.is_v0_p2wpkh() {
+        "v0_p2wpkh"
+    } else if script.is_v0_p2wsh() {
+        "v0_p2wsh"
+    } else if script.is_provably_unspendable() {
```

Comment:
Tapyrus doesn't support Segwit, so don't you need `v0_p2wpkh` and `v0_p2wsh`?

Reply:
Fixed at bf325a37891dc3d5321559ee454a644baa72eb06